### PR TITLE
Add external resources to sidebar, add app link to top

### PR DESCRIPTION
**Pull request by inahga**
I'm a bit disappointed that docusaurus sidebar configuration happens through code rather than markdown, so sidebar tweaks for pages that aren't autogenerated are out of the ability for non-technical folks to modify. It does offer remarkable customization though.

Mainly looking for feedback on whether there's additional resources we think would be helpful to link.

Supports https://github.com/divviup/janus-ops/issues/1004

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -72,6 +73,11 @@ const config = {
             position: 'left',
             label: 'Documentation',
           },
+          {
+            href: 'https://app.divviup.org',
+            label: 'App',
```
Comment (jbr) [inline docusaurus.config.js:78]: maybe "Divvi Up" instead of app?

File: sidebars.js [hunk 2]
```diff
@@ -13,21 +13,46 @@
...
+  docSidebar: [
+    {type: 'autogenerated', dirName: '.'},
     {
-      type: 'category',
-      label: 'Tutorial',
-      items: ['tutorial-basics/create-a-document'],
-    },
+      'Resources': [
+        {
+          'Client Library References': [
```
Comment (jbr) [inline sidebars.js:21]: We don't currently have a link for it, but [the https://github.com/divviup/divviup-api/tree/main/client] will have rustdocs, and the cli will have documentation somewhere (maybe a README?)

File: docusaurus.config.js (hunk 1 above)
Comment (inahga) [reply docusaurus.config.js:78]: If you think that's clearer. I was thinking `App` just to disambiguate whether that link would take you to `app.divviup.org` or `divviup.org`

File: sidebars.js (hunk 2 above)
Comment (inahga) [reply sidebars.js:21]: I can add a temporary link for the DU client. For CLI docs perhaps our best bet is just to use this repository?

Comment (jbr) [reply sidebars.js:21]: My only worry there is that it's hard enough to keep changes within a repo synced and even harder to keep documentation synced with an external repo

Comment (inahga) [reply sidebars.js:21]: That's fair. I'll put a link to https://github.com/divviup/divviup-api/tree/main/cli, which will show a README once we have one.

**Review by branlwyd — APPROVED** [review]

**Pull request merged**